packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/card.css

```diff
@@ -0,0 +1,19 @@
+.root {
+    display: grid;
+    font-size: 0.875rem;
+    row-gap: 0.5rem;
+    padding-top: 1rem;
+}
+
+.address {
+    display: grid;
+    gap: 0.5rem;
+    grid-template-areas:
+        '.'
+        'area';
```

Comment:
I think we can accomplish this vertical alignment without the use of `areas`. 

`grid-template-columns: 1fr;` should do the trick and allow you get rid of the added class within the address div.

Reply:
This accounts for multiple address rows without having to hard code any `address.length` logic.

Reply:
Oh you meant multiple `street` entries :) I gotcha.

Reply:
I'm having some grid critters flashbacks; maybe I could have done `grid-row-start: -1` instead.

Reply:
I don't even know why this was needed, the default flow works fine. Removed in 32ea1ca.